### ISSUE TITLE
compile errors in convert.cc

### ISSUE THREAD

**Issue by lilyanatia** [flightaware/dump978#10]
```
convert.cc: In static member function ‘static flightaware::uat::SampleConverter::Pointer flightaware::uat::SampleConverter::Create(flightaware::uat::SampleFormat)’:
convert.cc:45:20: error: ‘runtime_error’ is not a member of ‘std’
   45 |         throw std::runtime_error("format not implemented yet");
      |                    ^~~~~~~~~~~~~
convert.cc:44:5: error: control reaches end of non-void function [-Werror=return-type]
   44 |     default:
      |     ^~~~~~~
```

adding `#include <stdexcept>` seems to fix it.

**Comment by bovine** [comment]
What compiler version and OS distribution are you using?

**Comment by lilyanatia** [comment]
GCC 10.1.0 on Arch Linux.

**Comment by mutability** [comment]
What's your build environment?

**Comment by mutability** [comment]
Oops, missed the newer comments.

**Comment by Ai-rin** [comment]
Just stumbled upon this bug myself (why close?).
Confirm, that adding `#include <stdexcept>` fixes it.
gcc version 10.1.1 on openSUSE

**Comment by lilyanatia** [comment]
> why close?

it's fixed: 24a98ec

**Comment by Ai-rin** [comment]
Ah, in "dev" branch.
